### PR TITLE
Add 1.4.8 Post

### PR DESCRIPTION
**Pull request by basicDamen**

### CODE REVIEW

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+The !kc command now supports Barrows Chests and Chambers of Xeric kill counts.
+
+![barrows-kc](/img/blog/1.4.8-Release/barrows-kc.png)
+
+![raids-kc](/img/blog/1.4.8-Release/raids-kc.png)
+
+There are also several smaller improvements and bug fixes, including:
+ * Screenshots
+ 	* Raids screenshots will not be named "Chambers of Xeric"
```
Comment (Abextm) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: this is inverse of what the commit says

**Comment by Adam-** [comment]
The two killcounts were fixed in 1.4.7.2, and should not be included in this post probably

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+---
+title: '1.4.8 Release'
+description: 'Kill count command, prayer bars, and Tears of Guthix plugin'
+author: Damen
+---
+
+You can now filter through the Fairy Ring travel log upon configuration.
```
Comment (Nightfirecat) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: This should link to the person who authored this feature.

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+![fairy-ring-filter](/img/blog/1.4.8-Release/fairy-ring-filter.png)
+
+You can now pause skill timers upon log out or after a specified idle period.
```
Comment (Nightfirecat) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: This should link to the person who authored this feature.

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+![xp-tracker-pause](/img/blog/1.4.8-Release/xp-tracker-pause.png)
+
+The !kc command now supports Barrows Chests and Chambers of Xeric kill counts.
```
Comment (Nightfirecat) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: This should link to the person who authored this feature.

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+![raids-kc](/img/blog/1.4.8-Release/raids-kc.png)
+
+There are also several smaller improvements and bug fixes, including:
+ * Screenshots
+ 	* Raids screenshots will now be named "Chambers of Xeric"
+	* Hunter Level screenshots will now work properly
+	* Now utilizes a separate directory for DMM-related screenshots
+	* Now captures Theatre of Blood rewards
+ * Fairy Ring Info for the King Percival clue has been added
+ * Mythical Cape and Teak Kitchen Tables have been added to the Skill Calculator
```
Comment (Nightfirecat) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: "Mythical Cape" should be "Mounted Mythical Cape"

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 6]
```diff
@@ -4,28 +4,26 @@ description: 'Kill count command, prayer bars, and Tears of Guthix plugin'
...
+You can now filter through the Fairy Ring travel log upon configuration. (Thanks to Max Weber)
 
 ![fairy-ring-filter](/img/blog/1.4.8-Release/fairy-ring-filter.png)
 
-You can now pause skill timers upon log out or after a specified idle period.
+You can now pause skill timers upon log out or after a specified idle period. (Thanks to Levi Schuck)
 
 ![xp-tracker-pause](/img/blog/1.4.8-Release/xp-tracker-pause.png)
 
-The !kc command now supports Barrows Chests and Chambers of Xeric kill counts.
```
Comment (Nightfirecat) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: This entry is removed, but not mentioned in the bugfix section?

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+ * Mounted Mythical Cape and Teak Kitchen Tables have been added to the Skill Calculator
+ * Slayer NPCs that were highlighted though not actually there, no longer appear
+ * Ground Items should now show all items, seeing as they sometimes would not display everything
+ * Cacti are now successfully predicted regarding regrowth.
+ * Ring of Wealth now successfully updates the Boss Kill Counts.
+ 	* Barrows Chests and Raids Kill Counts are now working.
+ 
+ Enjoy!
+ 
+ - Damen
```
Comment (deathbeam) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: You should probably not use bullet point here

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+	* Hunter Level screenshots will now work properly
+	* Now utilizes a separate directory for DMM-related screenshots
+	* Automatically captures Theatre of Blood rewards
+ * Fairy Ring Info for the King Percival clue has been added
+ * Mounted Mythical Cape and Teak Kitchen Tables have been added to the Skill Calculator
+ * Slayer NPCs that were highlighted though not actually there, no longer appear
+ * Ground Items should now show all items, seeing as they sometimes would not display everything
+ * Cacti are now successfully predicted regarding regrowth.
+ * Ring of Wealth now successfully updates the Boss Kill Counts.
+ 	* Barrows Chests and Raids Kill Counts are now working.
```
Comment (deathbeam) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: Why is this on new level?

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
+---
+title: '1.4.8 Release'
+description: 'Kill count command, prayer bars, and Tears of Guthix plugin'
+author: Damen
+---
+
+You can now filter through the Fairy Ring travel log upon configuration. (Contributed by [@Max Weber (Abex)](https://github.com/Abextm))
```
Comment (deathbeam) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: We usually just put @<gh_user> tags here

File: src/_posts/2018-07-18-12-00-1.4.8-Release.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+![fairy-ring-filter](/img/blog/1.4.8-Release/fairy-ring-filter.png)
+
+You can now pause skill timers upon log out or after a specified idle period. (Contributed by [@Levi Schuck](https://github.com/LeviSchuck))
```
Comment (deathbeam) [inline src/_posts/2018-07-18-12-00-1.4.8-Release.md]: Same here

**Comment by Adam-** [comment]
Thanks for the initial work on this.

**Pull request closed without merging**